src/chatClient.cpp

```diff
@@ -1783,7 +1788,7 @@ ChatRoomList::~ChatRoomList()
 GroupChatRoom::GroupChatRoom(ChatRoomList& parent, const mega::MegaTextChat& aChat)
 :ChatRoom(parent, aChat.getHandle(), true, aChat.getShard(),
   (chatd::Priv)aChat.getOwnPrivilege(), aChat.getCreationTime()),
-  mHasTitle(false), mRoomGui(nullptr)
+  mHasTitle(false), mRoomGui(nullptr), mTitleIsDecrypting(true)
```

Comment:
I think the fix should simply change `mHasTitle(false)` by `aChat.getTitle()` to properly initialize mHasTitle` when adding chatrooms from API